ecosystem/resources/members/qclib.toml

```
@@ -0,0 +1,8 @@
+name = "Qclib"
+url = "https://github.com/qclib/qclib"
+description = "Qclib is a quantum computing library implemented using Qiskit. The focus of Qclib is on preparing quantum states, but it is not limited to that."
```

This needs to be <=135 characters. Is this change ok? The description should appear right underneath the project name.

```suggestion
description = "A quantum computing library implemented using Qiskit. The focus of Qclib is on preparing quantum states, but it is not limited to that."
```

---

I agree with the change. Having the description appear directly beneath the project name works well. Thank you!